proto_quic_tools/update.py

```diff
@@ -113,12 +113,18 @@ def copy_modified_files():
 
 # Since proto-quic users do not use gclient sync, runs necessary parts of it.
 def sync():
-  command = "/" + proto_quic_root + "/third_party/binutils/download.py"
+  command = "/" + proto_quic_root + "proto_quic_tools/sync.sh"
+  print "running", command;
+  os.system(command)
+
+def cleanup():
+  command = "/" + proto_quic_root + "proto_quic_tools/cleanup.sh"
   print "running", command;
   os.system(command)
 
 
-sync()
 copy_directories()
```

Comment:
Might be worthwhile to add some comments here about why we're doing things in this order.

copy, merge, copy, cleanup - attempts to leave the workspace in exactly the state it will look to someone checking out the post-commit repo

sync - pull down the required files which are not part of the repo